### DISCUSSION TITLE
Compile ERROR: Dependency "mpi" not found, tried pkgconfig, config-tool and system

### DISCUSSION THREAD

**Discussion by zhaokuohaha** [su2code/SU2#2115] [General]
**Describe the bug**
Compile MPI Version error on Windows

I can build successfully by run ` py .\meson.py build` ant `ninja.exe build install` , but it can't  compile with ` py .\meson.py build -Dwith-mpi=enabled`

the error message :
```
Version: 0.61.1
Source dir: F:\suflow\code\Su-FVMu-Solver
Build dir: F:\suflow\code\Su-FVMu-Solver\build
Build type: native build
Project name: SU2
Project version: 7.5.1 "Blackbird"
C compiler for the host machine: gcc (gcc 8.1.0 "gcc.exe (x86_64-posix-seh-rev0, Built by MinGW-W64 project) 8.1.0")
C linker for the host machine: gcc ld.bfd 2.30
C++ compiler for the host machine: c++ (gcc 8.1.0 "c++.exe (x86_64-posix-seh-rev0, Built by MinGW-W64 project) 8.1.0")
C++ linker for the host machine: c++ ld.bfd 2.30
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program python3 found: YES (C:\Users\zeng\AppData\Local\Programs\Python\Python37\python.exe)
WARNING: Python files installed by Meson might not be found by python interpreter.
 This warning can be avoided by setting "python.platlibdir" option.
WARNING: Python files installed by Meson might not be found by python interpreter.
 This warning can be avoided by setting "python.purelibdir" option.
Found pkg-config: C:\Program Files\mingw-w64\bin\pkg-config.EXE (0.26)
mpicc found: NO
Run-time dependency MPI for c found: YES
mpic++ found: NO
Run-time dependency MPI for cpp found: NO (tried pkgconfig, config-tool and system)

meson.build:61:2: ERROR: Dependency "mpi" not found, tried pkgconfig, config-tool and system

A full log can be found at F:\suflow\code\Su-FVMu-Solver\build\meson-logs\meson-log.txt
```

meson-log.txt: 
[meson-log.txt](https://github.com/su2code/SU2/files/12427739/meson-log.txt)

**Bug report checklist**

- [x] Consulting the build instructions (https://su2code.github.io/docs_v7/Build-SU2-Windows/).
- [x] Looking for similar problems on GitHub or CFD-Online (https://www.cfd-online.com/Forums/su2/).
- [x] Updating to the newest version of SU2 (either develop, master, or the pre-built executables https://su2code.github.io/download.html).

**Desktop (please complete the following information):**
 - OS: Windows 11
 - C++ compiler and version: gcc.exe (x86_64-posix-seh-rev0, Built by MinGW-W64 project) 8.1.0
 - MPI implementation and version: msmpi 10.1.2
 - SU2 Version: 7.5.1

**Comment by bigfooted** [comment]
I do not use windows, but have a look at this part of the installation manual: https://su2code.github.io/docs_v7/Build-SU2-Linux-MacOS/#configuration-and-compilation

locate the folder where mpic, mpicxx etc. are and add the folder to the environment variables  MPICC and MPICXX so they can be found.


> MPI installation is not found
> 
> Meson looks for an MPI installation using [pkg-config](https://en.wikipedia.org/wiki/Pkg-config). But if your MPI implementation does not provide them, it will search for the standard wrapper executables, mpic, mpicxx, mpic++. If these are not in your path, they can be specified by setting the standard environment variables MPICC, MPICXX during configuration.
